pkg/version/version.go

```diff
@@ -55,11 +55,17 @@ func String() string {
 }
 
 func GetBuild() *Build {
+	var b string
+	if BuildNumber == "" {
+		b = "0"
```

Comment:
Also, now that you point this out, where does BuildNumber get defined? It looks like this function should be a method w/ BuildNumber a receiver member.